backend/src/govsg/models/govsg-message-transactional.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { GovsgTemplate } from './govsg-template'
+import { GovsgMessageStatus } from '@core/constants'
+
+@Table({
+  tableName: 'govsg_messages_transactional',
+  underscored: true,
+  timestamps: true,
+})
+export class GovsgMessageTransactional extends Model<GovsgMessageTransactional> {
+  @ForeignKey(() => GovsgTemplate)
+  @Column(DataType.BIGINT)
+  templateId: number
+
+  @BelongsTo(() => GovsgTemplate)
+  template: GovsgTemplate
+
+  @Column(DataType.STRING)
+  recipient: string
+
+  @Column(DataType.JSONB)
+  params!: object
+
+  @Column(DataType.STRING)
+  serviceProviderMessageId?: string
```

Comment:
what is this supposed to map to? is it:
- SMS vs WhatsApp; or
- the specific WhatsApp client (e.g. phone number 1 vs phone number 2)